app/javascript/utils/form/final_form/Field.js

```diff
@@ -99,7 +99,7 @@ export const SelectField = ({ fieldName, label, blockNote, validation, id, optio
             }}
             id={id || `form-${fieldName}`}
             name={input.name}
-            className={classNames(className || 'form-group', (meta.error && meta.touched && 'error') || '')}>
+            className={classNames(className, (meta.error && meta.touched && 'error') || '')}>
```

Comment:
this fixed row spacing issues from https://www.pivotaltracker.com/story/show/166909855